config/telegramBotHandler.js

```diff
@@ -36,86 +36,86 @@ class TelegramBotHandler {
         });
 
         // Handle the /new_lottery command
-        this.bot.onText(/\/new_lottery/, async (msg) => {
-            const chatId = msg.chat.id;
-            const username = msg.from.username;
-            this.lotteries[chatId] = new Lottery(chatId, username);
-            this.lotteries[chatId].reset(chatId);
-            this.lotteries[chatId].sendMessage(this.startMessage, keyboards);
-            // Send the "hello world" message when /new_lottery is triggered
-            this.commandStatus[chatId] = 'new_lottery';
-        });
-        this.bot.on('callback_query', async (query) => {
-            const chatId = query.message.chat.id;
-            const data = query.data;
-            const lottery = this.lotteries[chatId];
+        // this.bot.onText(/\/new_lottery/, async (msg) => {
+        //     const chatId = msg.chat.id;
+        //     const username = msg.from.username;
+        //     this.lotteries[chatId] = new Lottery(chatId, username);
+        //     this.lotteries[chatId].reset(chatId);
+        //     this.lotteries[chatId].sendMessage(this.startMessage, keyboards);
+        //     // Send the "hello world" message when /new_lottery is triggered
+        //     this.commandStatus[chatId] = 'new_lottery';
+        // });
+        // this.bot.on('callback_query', async (query) => {
+        //     const chatId = query.message.chat.id;
+        //     const data = query.data;
+        //     const lottery = this.lotteries[chatId];
 
-            switch(data){
```

Comment:
Please fix this part.
Switch status is not correct.

Reply:
I fixed.

Reply:
I fixed.